azurerm/internal/services/containers/kubernetes_nodepool.go

```diff
@@ -102,15 +102,6 @@ func SchemaDefaultNodePool() *schema.Schema {
 					},
 				},
 
-				"node_taints": {
-					Type:     schema.TypeList,
-					ForceNew: true,
-					Optional: true,
-					Elem: &schema.Schema{
-						Type: schema.TypeString,
-					},
-				},
```

Comment:
this'd be a breaking change - can we revert this and deprecate this field (with a message) and error if this is set during creation?